src/components/Router/index.js

```diff
@@ -153,6 +158,7 @@ class HistoryRouter extends Component {
   render = (props) =>
       <StepsRouter {...props}
         componentsList={this.componentsList()}
+        shouldShowComplete={this.shouldShowComplete}
```

Comment:
where is this called?

Reply:
Good spot, forgot it there when refactoring